src/com/kinvey/sample/kitchensink/account/RegisterFragment.java

```diff
@@ -207,13 +210,11 @@ private boolean validateInput() {
     public void onClick(View v) {
         if (v == registerButton){
             submit();
-
         }
-
     }
 
     public void submit() {
-        getClient().user().create(userName.getText().toString(), password.getText().toString(), new KinveyUserCallback() {
+    	((KitchenSinkApplication) getSherlockActivity().getApplicationContext()).getClient().user().create(userName.getText().toString(), password.getText().toString(), new KinveyUserCallback() {
```

Comment:
why not use the getClient() method?  It does the same thing, just less code to copy/paste and it's more readable?